integrations/docker/images/chip-build-mbedos/Dockerfile

```diff
@@ -0,0 +1,58 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+# ------------------------------------------------------------------------------
+# Install system tools via apt
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+    python3-setuptools \
+    python3-usb \
+    software-properties-common \
+    build-essential \
+    astyle \
+    mercurial \
+    && rm -rf /var/lib/apt/lists \
+    && : # last line
+
+# ------------------------------------------------------------------------------
+# Install Mbed-OS sources
+RUN set -x \
+    && (mkdir -p /opt/mbed-os \
+    && cd /opt/mbed-os \
+    && git clone --progress -b mbed-os-6.6.0 https://github.com/ARMmbed/mbed-os.git ./) \
```

Comment:
The git repository is quite large, would it be possible to download the release from github instead ?

Reply:
Yes, I see now - we can download compressed sources (in .tar.gz) from github release page of Mbed OS.
Thanks, good point!

Reply:
Done.